client_test.go

```diff
@@ -3,17 +3,23 @@ package supabase_test
 import (
 	"fmt"
 	"testing"
+	"time"
 
-	"github.com/supabase-community/supabase-go"
+	"github.com/supabase-community/auth-go/types"
+
+	supa "github.com/supabase-community/supabase-go"
 )
 
 const (
-	API_URL = "https://your-company.supabase.co"
-	API_KEY = "your-api-key"
+	API_URL       = "https://your-company.supabase.co"
+	API_KEY       = "your-api-key"
+	TEST_EMAIL    = "test@example.com"
+	TEST_PASSWORD = "test123456"
+	TEST_PHONE    = "+1234567890"
 )
 
 func TestFrom(t *testing.T) {
-	client, err := supabase.NewClient(API_URL, API_KEY, nil)
```

Comment:
Why prefer `supa` over `supabase`? The latter feels better to me.

Reply:
You're right. It was to quickly distinguish my old closet from a new revision.